packages/synapse-react-client/src/components/PortalFeaturedPartners/PortalFeaturedPartners.tsx

```diff
@@ -0,0 +1,217 @@
+    enabled: !!fileId,
+  })
+  const dataUrl = stablePresignedUrl?.dataUrl
+
+  return dataUrl
+}
+
+const RenderPartnerIcon = ({
+  partner,
+  imageColIndex,
+  organizationNameColIndex,
+  websiteColIndex,
+  entityId,
+  isLoading,
+}: Icons) => {
+  const fileId = partner.values[imageColIndex] ?? ''
+  const imageUrl = useImageUrl(fileId ?? '', entityId)
+
+  if (!imageUrl && !partner?.values[organizationNameColIndex]) {
+    return
+  }
+
+  return isLoading ? (
+    <Skeleton variant="rectangular" height={101} width={156} />
+  ) : (
+    <Link
+      href={partner.values[websiteColIndex] ?? ''}
+      target="_blank"
+      sx={{
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+        textDecoration: 'none',
+        '&:hover': { textDecoration: 'none' },
+      }}
+    >
+      {imageUrl ? (
+        <CardMedia
+          component="img"
+          image={imageUrl}
+          aria-hidden="true"
+          sx={{
+            width: '156px',
+            height: '101px',
+            objectFit: 'contain',
+            borderRadius: '10px',
+          }}
+        />
+      ) : (
+        <Box
+          sx={{
+            display: 'flex',
+            width: '165px',
+            padding: '3px',
+            flexDirection: 'column',
+            gap: '10px',
+            border: '2px solid',
+            borderColor: 'grey.900',
+            backgroundColor: '#FFF',
+            textAlign: 'center',
+          }}
+        >
+          <Box
+            sx={{
+              display: 'flex',
+              padding: '10px',
+              justifyContent: 'center',
+              alignItems: 'center',
+              gap: '10px',
+              border: '1px solid',
+              borderColor: 'grey.900',
+            }}
+          >
+            <Typography
+              variant="headline3"
+              sx={{
+                fontSize: '14px',
+                fontWeight: 400,
+                lineHeight: '130%',
+                color: 'grey.900',
+              }}
+            >
+              {partner?.values[organizationNameColIndex ?? -1]}
+            </Typography>
+          </Box>
+        </Box>
+      )}
+    </Link>
+  )
+}
+
+const PortalFeaturedPartners = ({ sql }: PortalFeaturedPartnersProps) => {
+  const entityId = parseEntityIdFromSqlStatement(sql)
+
+  const queryBundleRequest: QueryBundleRequest = {
+    partMask:
+      SynapseConstants.BUNDLE_MASK_QUERY_SELECT_COLUMNS |
+      SynapseConstants.BUNDLE_MASK_QUERY_RESULTS,
+    concreteType: 'org.sagebionetworks.repo.model.table.QueryBundleRequest',
+    entityId,
+    query: {
+      sql,
+    },
+  }
+  const { data: queryResultBundle, isLoading } =
+    useGetQueryResultBundle(queryBundleRequest)
+
+  const dataRows = queryResultBundle?.queryResult!.queryResults.rows ?? []
+
+  enum ExpectedColumns {
+    CARDLOGO = 'cardLogo',
+    ORGANIZATIONNAME = 'organizationName',
+    WEBSITE = 'website',
```

Comment:
Minor change for readability
```suggestion
    CARD_LOGO = 'cardLogo',
    ORGANIZATION_NAME = 'organizationName',
    WEBSITE = 'website',
```